src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,135 @@
+/**
+* @title WET-BOEW Exit script plugin
+* @overview Plugin redirects users to non secure site
+* @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+* @author @ipaksc
+*/
+( function( $, window, wb ) {
+"use strict";
+var componentName = "wb-exitscript",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	exiturlparam = componentName + "-urlparam",
+	i18nDict = {
+		en: {
+			"msgboxHeader": "Warning",
+			"exitMsg": "You are about to leave a secure site, do you wish to continue?",
+			"yesBtn": "Yes",
+			"cancelBtn": "Cancel"
+
+		},
+		fr: {
+			"msgboxHeader": "Avertissement",
+			"exitMsg": "Vous êtes sur le point de quitter un site sécurisé. Voulez-vous continuer?",
+			"yesBtn": "Oui",
+			"cancelBtn": "Annuler"
+
+		}
+	},
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			settings,
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+				true,
+				window[ componentName ],
+				wb.getData( $elm, componentName )
+
+			);
+
+			$elm.data( componentName, settings );
+
+			i18nDict = i18nDict[ wb.lang || "en" ];
+
+			if ( $elm.hasClass( exiturlparam ) ) {
+				this.outerHTML = "<a href='" + sessionStorage.getItem( "originalURL" ) + "'>" + sessionStorage.getItem( "originalURL" ) + "</a>";
+				if ( document.querySelector( selector + "-targetparam" ) ) {
```

Comment:
It is preferable to keep it in scope of the element instead of getting the first instance in the page.

Reply:
@duboisp I need to discuss this with Stéphane. Maybe I will remove   if ( document.querySelector( selector + "-targetparam" ) ) {  from the script.